snappy/click.go

```diff
@@ -236,7 +236,7 @@ func generateSnapServicesFile(app *AppYaml, baseDir string, aaProfile string, m
 
 	return systemd.New(dirs.GlobalRootDir, nil).GenServiceFile(
 		&systemd.ServiceDescription{
-			AppName:        m.Name,
+			SnapName:       m.Name,
 			ServiceName:    app.Name,
```

Comment:
Yep, I'd like to clean this area up a lot but do so in small trivial-to-review changes that hopefully don't break tests. This can easily explode past "I see what the diff is about" since many changes can easily affect tons of test data.